crates/iota-node/src/main.rs

```diff
@@ -72,8 +73,9 @@ fn main() {
 
     let runtimes = IotaRuntimes::new(&config);
     let metrics_rt = runtimes.metrics.enter();
-    let registry_service = iota_metrics::start_prometheus_server(config.metrics_address);
+    let mut registry_service = iota_metrics::start_prometheus_server(config.metrics_address);
```

Comment:
Is `mut` really necessary here? Other metrics were registered without this keyword.